README.md

```diff
@@ -15,16 +15,19 @@ var token = '{YOUR_API_V2_TOKEN}';
 var auth0 = require('auth0')(token);
 ~~~
 
-### app_metadata
+### User's metadata
+Auth0 allows use to store data about a user that does not come from the user's identity provider. This is known as "metadata". There are two types of metadata: `user_metadata` and `app_metadata`. You can find out more [here](https://auth0.com/docs/apiv2Changes#8).
+
+#### app_metadata
 To update the value of a property or create it if it does not exist:
 ~~~js
 var user_id = '...'
 auth0.users(user_id).appMetadata.update({
   roles: ['reader']
-}, function(err, user){
+}).then(function(user){
```

Comment:
Also, mention that it also supports callback and link to some example from the `examples/` folder